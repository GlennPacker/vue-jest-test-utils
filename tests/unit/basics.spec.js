import { mount, shallowMount } from '@vue/test-utils';
import HelloWorld from '@/components/HelloWorld.vue';

describe('basic examples -', () => {
    it('renders / mount', () => {
        const wrapper = mount(HelloWorld)
        const container = wrapper.find('.container');
        expect(container.exists()).toBe(true)
    })

    it('shallow mount', () => {
        const wrapper = shallowMount(HelloWorld);

        expect(wrapper.exists());
        expect(wrapper.html()).toMatchSnapshot()
    })
})