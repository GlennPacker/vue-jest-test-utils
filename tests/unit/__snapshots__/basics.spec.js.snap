// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic examples - shallow mount 1`] = `
<div class="container">
  <h1>Hello </h1>
  <div>hi </div>
</div>
`;
